src/aggregator/server/rawtcp/server_test.go

```diff
@@ -249,42 +221,34 @@ func testRawTCPServerHandleUnaggregated(
-				require.NoError(t, encoder.EncodeGaugeWithPoliciesList(testGaugeWithPoliciesList))
-				stream = encoder.Encoder().Bytes()
-			case protobufEncoding:
-				encoder := protobuf.NewUnaggregatedEncoder(protobuf.NewUnaggregatedOptions())
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type:                 encoding.CounterWithMetadatasType,
-					CounterWithMetadatas: testCounterWithMetadatas,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type: encoding.BatchTimerWithMetadatasType,
-					BatchTimerWithMetadatas: testBatchTimerWithMetadatas,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type:               encoding.GaugeWithMetadatasType,
-					GaugeWithMetadatas: testGaugeWithMetadatas,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type: encoding.TimedMetricWithMetadataType,
-					TimedMetricWithMetadata: testTimedMetricWithMetadata,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type: encoding.PassthroughMetricWithMetadataType,
-					PassthroughMetricWithMetadata: testPassthroughMetricWithMetadata,
-				}))
-				require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
-					Type: encoding.ForwardedMetricWithMetadataType,
-					ForwardedMetricWithMetadata: testForwardedMetricWithMetadata,
-				}))
-				buf := encoder.Relinquish()
-				stream = buf.Bytes()
-			}
+			encoder := protobuf.NewUnaggregatedEncoder(protobuf.NewUnaggregatedOptions())
+			require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
+				Type:                 encoding.CounterWithMetadatasType,
+				CounterWithMetadatas: testCounterWithMetadatas,
+			}))
+			require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
+				Type:                    encoding.BatchTimerWithMetadatasType,
+				BatchTimerWithMetadatas: testBatchTimerWithMetadatas,
+			}))
+			require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
+				Type:               encoding.GaugeWithMetadatasType,
+				GaugeWithMetadatas: testGaugeWithMetadatas,
+			}))
+			require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
+				Type:                    encoding.TimedMetricWithMetadataType,
+				TimedMetricWithMetadata: testTimedMetricWithMetadata,
+			}))
+			require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
+				Type:                          encoding.PassthroughMetricWithMetadataType,
+				PassthroughMetricWithMetadata: testPassthroughMetricWithMetadata,
+			}))
+			require.NoError(t, encoder.EncodeMessage(encoding.UnaggregatedMessageUnion{
+				Type:                        encoding.ForwardedMetricWithMetadataType,
+				ForwardedMetricWithMetadata: testForwardedMetricWithMetadata,
+			}))
+			buf := encoder.Relinquish()
+			stream = buf.Bytes()
```

Comment:
nit:

```lang=go
var (
  buf.   = encoder.Relinquish()
  stream = buf.Bytes()
)
```

(although I don't think the intermediate vars really provide a ton of value, so just using `encoder.Relinquish().Bytes()` directly is probably fine below as well)